.github/workflows/code_style.yml

```diff
@@ -24,20 +24,19 @@ jobs:
           cmake ..
 
       - name: Check code style
-        run: make style-check
-        working-directory: build
+        run: cmake --build build --target clang_format_check_all
 
       - name: Create code style diff
         if: failure()
         run: |
-          ngraph/maint/apply-code-format.sh
-          git diff >ngraph_code_style_diff.patch
+          cmake --build build --target clang_format_fix_all
+          git diff > code_style_diff.patch
 
       - uses: actions/upload-artifact@v2
         if: failure()
         with:
-          name: ngraph_code_style_diff
-          path: ngraph_code_style_diff.patch
+          name: code_style_diff
+          path: code_style_diff.patch
```

Comment:
As above - `.diff` vs `.patch`

Reply:
It is a legacy extension 😄 But yes, I agree that we can rename it.

Reply:
Fixed